### DISCUSSION TITLE
How things are suposed to work! At least for now ...

### DISCUSSION THREAD

**Discussion by coolapso** [streamdeck-linux-gui/streamdeck-linux-gui#41] [Announcements]
After reading multiple discussions here and chats on the Discord server, I genuinely believe that there are some misconceptions around here. Many individuals don't seem to grasp how open-source and community-maintained projects actually function.

I find [this section of the contribution guide](https://streamdeck-linux-gui.github.io/streamdeck-linux-gui/contributing/contributing-guide/#making-a-contribution) very clear. However, let me clarify a few points regardless. Or at the very least, let me explain how things work around here for now. This doesn't imply that changes can't occur in the future.

As a community project of modest size and with no funds, where functionality and maintenance stem purely from unpaid voluntary contributions, the notion of "assigning" issues or "tickets," or whatever term you prefer, is nonexistent. Similarly, there's no concept of a team or individuals being accountable for anything. If anyone is currently responsible for anything here, it's me.

In my role as a project maintainer, I'm here to guide people, ensure the project's health and longevity, review pull requests, adhere to standards, and ultimately facilitate proper merging.

When it comes to features, those who desire specific ones can request them [here](https://github.com/streamdeck-linux-gui/streamdeck-linux-gui/discussions/categories/ideas-feature-requests). Users then vote on these requests, and anyone with the coding skills and the desire to contribute can tackle whatever they feel inclined to. They might even opt for the feature with the fewest votes. But who am I, or anyone else on GitHub, to dictate what individuals should do or prioritize with their own free time?

Regarding bugs, the process is quite similar to that of features. Those willing to address certain bugs can simply go ahead and do so. There's one exception, though. If a bug is so critical or severe that it completely disrupts the application, rest assured that I will ensure it's fixed, or at the very least, I will appeal to the community for assistance.

Moreover, with regards to the concept of a "team" or "individuals," for now, I am the sole entity responsible. I would appreciate having someone to whom I could grant similar privileges within the project. However, I won't bestow such privileges on the first person who expresses interest. As a software that's widely used in the Linux community, project maintainers should possess a certain level of competence. They should be reasonably proficient in Python, Git, and GitHub (yes, they aren't the same!), and they should also grasp the functioning of pipelines. Essentially, they should understand how to keep the project operational, with Python ironically being the least essential skill.

I hope this sheds light on how things are expected to proceed. I welcome suggestions and comments from all of you and am eager to hear your thoughts.

**Comment by Danie10** [comment]
@4s3ti no you are dead right. And we need to remember this is also a new fork that has lost its original maintainer. An original maintainer is often an active driving force pushing out new features etc. This project will need to consolidate around who the chief contributors may be, and hopefully attract more devs. But it is up to anyone to decide what to tackle - there is no right of anyone to make demands. I think we're all thankful you took on the coordinating role though.

So any specific bugs or stability issues would ideally be a primary concern, but it is up to feature requestors to motivate their feature so that it catches the attention of any devs, who may also see value in implementing a new feature. Feature requestors should also steer clear of duplicating requests (search and look at other discussions first) and possibly include requests in existing discussions IF it actually relates to the same request, or naturally may go with it.

Hopefully your post will manage any expectations by everyone.
